packages/functional-tests/src/tests/gltf-gen.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+import * as GltfGen from '@microsoft/gltf-gen';
+import App from '../app';
+import Server from '../server';
+import delay from '../utils/delay';
+import destroyActors from '../utils/destroyActors';
+import Test from './test';
+import { resolve } from 'path';
+
+export default class GltfGenTest extends Test {
+
+    constructor(app: App, private baseUrl: string) {
+        super(app);
+    }
+
+    public async run(): Promise<boolean> {
+        let spherePrim = new GltfGen.Sphere(0.5);
```

Comment:
Use const here. Did tslint not flag this? Wondering if lint is not working for you on the functional-tests project.